test/check-application

```diff
@@ -1853,6 +1859,82 @@ class TestApplication(testlib.MachineCase):
+        b.set_input_text('#run-image-healthcheck-start-period input', '5')
+        b.click('#run-image-healthcheck-retries button:nth-child(1)')
+        b.wait_val("#run-image-healthcheck-retries input", 2)
+        b.click('.pf-c-modal-box__footer button:contains("Create and run")')
+
+        self.waitContainerRow("healthy")
+
+        healthy_sha = self.execute(auth, "podman inspect --format '{{.Id}}' healthy").strip()
+        self.waitContainer(healthy_sha, auth, state='RunningHealthy')
+
+        self.toggleExpandedContainer("healthy")
+        b.click(".pf-m-expanded button:contains('Health check')")
+
+        b.wait_in_text('#container-details-healthcheck dt:contains("Command") + dd', 'true')
+        b.wait_in_text('#container-details-healthcheck dt:contains("Interval") + dd', '325 seconds')
+        b.wait_in_text('#container-details-healthcheck dt:contains("Retries") + dd', '2')
+        b.wait_in_text('#container-details-healthcheck dt:contains("Timeout") + dd', '35 seconds')
+        b.wait_in_text('#container-details-healthcheck dt:contains("Start period") + dd', '5 seconds')
+        b.wait_not_present('#container-details-healthcheck dt:contains("Failing streak")')
+
+        self.assertEqual(self.execute(auth, "podman inspect --format '{{.Config.Healthcheck}}' healthy").strip(), "{[true] 5s 5m25s 35s 2}")
+
+        b.wait_in_text(".ct-listing-panel-body tbody tr", "Passed health run")
+        b.wait_not_present(".ct-listing-panel-body tbody tr:nth-child(2)")
+        b.wait_visible(".ct-listing-panel-body tbody:nth-child(2) svg.green")
+
+        # Trigger run manually
+        self.performContainerAction("healthy", "Run health check")
+        b.wait_visible(".ct-listing-panel-body tbody tr:nth-child(2)")
+
+        self.toggleExpandedContainer("healthy")
+
+        self.execute(auth, "podman run --name sick -dt --health-cmd false --health-interval 5s busybox")
+        self.waitContainerRow("sick")
+        unhealthy_sha = self.execute(auth, "podman inspect --format '{{.Id}}' sick").strip()
+        self.waitContainer(unhealthy_sha, auth, state='RunningUnhealthy')
+        # Unhealthy should be first
+        b.wait_collected_text("#containers-containers .container-name", "sickhealthy")
+
+        self.toggleExpandedContainer("sick")
+        b.click(".pf-m-expanded button:contains('Health check')")
+        b.wait_visible(".pf-m-expanded .ct-listing-panel-body tbody tr:nth-child(1)")
+        b.wait_visible(".pf-m-expanded .ct-listing-panel-body tbody tr:nth-child(4)")
+        b.wait_visible(".pf-m-expanded .ct-listing-panel-body tbody tr:nth-child(2) svg.red")
+        b.wait_visible('#container-details-healthcheck dt:contains("Failing streak")')
+        self.assertGreater(int(b.text('#container-details-healthcheck dt:contains("Failing streak") + dd')), 3)
+
+    @testlib.skipImage("Ubuntu-stable does not get healt check run signals", "ubuntu-stable")
```

Comment:
health

Reply:
Is there an issue somewhere about this or is ubuntu-stable using an older podman?

Reply:
yeah, I did not report this as this is related to just stable version having older podman